services/pkg/client/alchemy/alchemy_client.go

```diff
@@ -123,20 +123,38 @@ func (c *Client) NewRelayTransactor(ctx context.Context) (*bind.TransactOpts, er
 }
 
 // [{"forking": {"jsonRpcUrl": "httKBrdf", "blockNumber": 14000000}}
-func (c *Client) Reset(ctx context.Context, remoteURL string, remoteBlockNumber uint64) (uint64, error) {
+func (c *Client) Reset(ctx context.Context, remoteURL string, remoteBlockNumber uint64) (*json.RawMessage, error) {
 	forking := map[string]interface{}{
 		"jsonRpcUrl":  remoteURL,
 		"blockNumber": remoteBlockNumber,
 	}
 	params := map[string]interface{}{
 		"forking": forking,
 	}
-	var hex hexutil.Uint64
-	err := c.rpc.CallContext(ctx, &hex, "anvil_reset", params)
+	var res json.RawMessage
+	err := c.rpc.CallContext(ctx, &res, "anvil_reset", params)
```

Comment:
Presumably making calls like anvil_reset and anvil_mine are specific to Foundry tools and these functions will need OP Stack (geth?) versions when we move?

Reply:
yes they are specific to anvil ... but anvil would always be used as the simulation fork here, regardless of upstream chain ... so they would/could stay as is